File: maxhanna.client/src/app/daily-mining-earnings.ts
export class DailyMiningEarnings {
  date: Date;
  algos: number[];
  totalEarnings: number;

  constructor(date: Date, algos: number[], totalEarnings: number) {
    this.date = date;
    this.algos = algos;
    this.totalEarnings = totalEarnings;
  }
}

File: maxhanna.Server/Controllers/DataContracts/DailyMiningEarnings.cs
namespace maxhanna.Server.Controllers.DataContracts
{
    public class DailyMiningEarnings
    {
        public DateTime Date { get; set; }
        public List<int>? Algos { get; set; }
        public decimal TotalEarnings { get; set; }
    }
}
